docker/openemr/flex-3.17/opcache.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -e
+
+if [ ! "$OPCACHE_ON" == 1 ]; then
+   echo bad context for opcache.sh launch
+   exit 1
+fi
+
+if [ ! -f /etc/php-opcache-configured ]; then
+    # install opcache library
+    apk update
+    apk add --no-cache php81-opcache 
```

Comment:
1. would simply install php81-opcache when install all the other packages in Dockerfile?
2. are any of those settings in the default php.ini file? (if so, then could set them there since opcache is always going to be on)?
3. would it make sense to put this in 7.0.2 first (the dev docker that is build nightly from dev) and then do 7.0.1 when sure it is working well since the patch and rebuild of the 7.0.1 docker will be happening very, very soon :) (if testing well on your side, then ok to proceed with 7.0.1 also though)

Reply:
Yea, I think it makes sense to put it in 7.0.2, was targeting the dev dockers initially and had the thought that we'd want to put it in 7.0.1 but its probably too risky to push this in since we haven't updated the dockers yet.